Dockerfile.dev

```diff
@@ -44,7 +44,9 @@ RUN mv /app/database-docker/db/ /app/ && rm -rf /app/database-docker/
 RUN echo "host all  all    0.0.0.0/0  md5" >> /etc/postgresql/15/main/pg_hba.conf
 RUN echo "listen_addresses='*'" >> /etc/postgresql/15/main/postgresql.conf
 RUN sed -i "s/^port = .*/port = ${DB_PORT}/" /etc/postgresql/15/main/postgresql.conf
-RUN npm install --global yarn --force
+RUN curl -fsSL https://deb.nodesource.com/setup_18.x | bash - \
```

Comment:
I am not sure whether this was required or not. As the base image in this dockerfile is `middlewareeng/oss-base`, and I suppose this image is being built from Dockerfile.prebuilt(Where I upgraded to node 18)
If my understanding of the code is correct then changes to this file can be removed and a new commit can be made to `middlewareeng/oss-base` with the updates in the Dockerfile.prebuilt

I am not so sure about my understanding of this

Reply:
> I am not sure whether this was required or not. As the base image in this dockerfile is `middlewareeng/oss-base`, and I suppose this image is being built from Dockerfile.prebuilt(Where I upgraded to node 18) If my understanding of the code is correct then changes to this file can be removed and a new commit can be made to `middlewareeng/oss-base` with the updates in the Dockerfile.prebuilt
> 
> I am not so sure about my understanding of this

yep. Not needed here